spec/views/volunteer_ops/popup.html.erb_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-describe "volunteer_ops/_popup.html.erb", :type => :view do
+describe "volunteer_ops/_popup_harrow.html.erb", :type => :view do
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)